src/features/layout/nav/Nav.tsx

```diff
@@ -1,24 +1,100 @@
+import { AiOutlineClose, AiOutlineMenu } from 'react-icons/ai';
+import { BiUser } from 'react-icons/bi';
+import { BsCart3 } from 'react-icons/bs';
 import { NavLink } from 'react-router-dom';
+import { useState } from 'react';
+import { useTranslation } from 'react-i18next';
 
-const navStyle = {
-  background: 'orange',
-  display: 'flex',
-  margin: '0 auto',
-  height: '4rem',
-  width: '80%',
-  justifyContent: 'space-between',
-  padding: '1rem 1rem',
-};
+import Backdrop from './Backdrop';
+import LanguageButtons from './LanguageButtons';
+import NavBar from './NavBar';
+import logo from '../../../assets/s-logo.svg';
+import { navBurgerStyles, navStyles } from './navStyles';
 
 const Nav = () => {
+  const { t, i18n } = useTranslation('mainNav');
+
+  const changeLanguage = (lng: string) => {
+    i18n.changeLanguage(lng);
+  };
+  const [nav, setNav] = useState(false);
+
+  const handleNav = () => {
+    setNav(!nav);
+  };
   return (
-    <div className="Nav" style={navStyle}>
-      <NavLink to="/"> HomePage </NavLink>
-      <NavLink to="/about-us"> AboutUs </NavLink>
-      <NavLink to="/contact"> ContactPage </NavLink>
-      <NavLink to="/user"> UserPage </NavLink>
-      <NavLink to="/product-series"> ProductSeriesPage </NavLink>
-      <NavLink to="/admin"> AdminPage </NavLink>
+    <div className="flex justify-between items-center h-20 sm:h-36 md:h-44 max-w-7xl mx-auto sm:px-3 md:px-6 lg:px-8">
+      <div
+        className={
+          !nav ? 'p-6 sm:p-3 lg:p-4 w-36 sm:w-48  md:w-64 lg:w-72' : 'hidden'
+        }
+      >
+        <img src={logo} alt="smallets logo" width="100%" height="100%" />
+      </div>
+      <section className="hidden sm:block">
+        <div className="flex p-4 pl-8 gap-8 justify-end text-2xl">
+          <NavLink to="/login">
+            <BiUser />
+          </NavLink>
+          <NavLink to="/cart">
+            <BsCart3 />
+          </NavLink>
```

Comment:
These two links, and the next row of links, change the location of the nav links, they jump a bit (to reproduce: click on cart and then on products, and then on cart again. the text changes position

Reply:
Fixed. Thanks for the comment.